astroquery/rms/core.py

```diff
@@ -0,0 +1,329 @@
+ # put all imports organized as shown below
+ # 1. standard library imports
+
+# urlparse package just to Python 2.x
+from bs4 import BeautifulSoup
+import urllib
+import requests
+import urlparse
+import pandas as pd
+
+# 2. third party imports
+
+import astropy.units as u
+
+@async_to_sync
+class RMS_by_coordinates(BaseQuery):
+  """
+  Class to The RMS Cone Search by COORDINATE
+
+  Required:
+  
+  coordinates (19:27:42.1 16:37:25.0) ; [string] ; coordinates
+  
+  Inputs with default value:
+  
+  Search radius (arcsec) ;  [float]  ; por defecto 60 ; radius
+  Radio button           ;  [integer in range[1,4] ]: ; list_id
+    - Final RMS   ; value = 1 selected 
+    - Near/Mid-IR ; value = 2
+    - Rejected    ; value = 3
+    - MSX Colour  ; value = 4 
+  """
+  request_payload['text_field_1'] = coordinates
+  request_payload['radius_field'] = radius
+  request_payload['listID']       = list_id
+  
+  session       = requests.Session()
+  base_response = S.get('http://rms.leeds.ac.uk/cgi-bin/public/RMS_CONE_SEARCH.cgi')
+  
+  result_send_query = session.post('http://rms.leeds.ac.uk/cgi-bin/public/RMS_SEARCH_RESULTS.cgi',data=request_payload)
+  
+  resut_soup = BeautifulSoup(result_send_query.text,'html.parser')
+  
+  # Create a list with the column names of the records table
+  columns = [th.text.strip() for th in resut_soup.find_all('th')][:-1]
+  result_query = soup.find("div",{"id":"content"}).find_all_next('form')
+  
+  df_query = pd.DataFrame.from_dict({i:scrape_forms(form) for i,form in enumerate(res)},orient='index')
```

Comment:
To be consistent with other modules, and as an official Astropy affiliated package, I would like to see usage of `astropy.table.Table` rather than `pandas`. Is there something in `pandas` you need here that `Table` cannot provide?